src/NDS.cpp

```diff
@@ -66,6 +66,7 @@ ARMv5* ARM9;
 ARMv4* ARM7;
 
 u32 NumFrames;
+u32 FramesSinceBoot;
```

Comment:
FramesSinceBoot seems reduandant to me, you use NumFrames too.

Btw both seems not be reset when loading a new game. I think for NumFrames it was an oversight from Arisotura.

Reply:
It is redundant in this code, but not in the DSHawk branch. There, NumFrames is what's reported to the client, and can be reset by the client; any frame can be arbitrarily marked as "frame 0". But this should not affect the RTC.

Neither are reset when loading a ROM, since a power cycle does not mean resetting the session. (And currently using the power button [I have more code for this on the DSHawk branch.] to power cycle just re-loads the ROM.) There must be some way to reboot the system without resetting any of these frame counters.

Reply:
ah I agree, I'll communicate that with Arisotura, we'll see what we can do with the current frontend refactor